.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "extends": "airbnb-base",
```

Comment:
airbnb?

Reply:
Ha! It was already in there - I've replaced the .yaml file with a .js file. `airbnb-base` seems to be a pretty standard thing to extend for linting rules. Not sure why, but perhaps a big part of it is that they open-sourced their lint files and they seem fairly sensible.